addons/external_intercom/script_component.hpp

```diff
@@ -0,0 +1,18 @@
+#define COMPONENT external_intercom
+#include "\z\tfar\addons\core\script_mod.hpp"
+
+#ifdef DEBUG_ENABLED_EXTERNAL_INTERCOM
```

Comment:
In Arma 2.06 you can set preprocessor macros via start parameter
https://community.bistudio.com/wiki/Arma_3:_Startup_Parameters
preprocDefine
could use this here to dynamically enable intercom debugging

Reply:
Not too sure how that all works? This was just a copy/paste from one of the other addons

Reply:
Oh, I see what you mean. You're saying I can set this variable to true in my launcher for additional logging. Thanks for the heads up!